Ginger/Ginger/App.xaml.cs

```diff
@@ -244,6 +244,14 @@ private void Application_Startup(object sender, StartupEventArgs e)
         {            
             Amdocs.Ginger.CoreNET.log4netLib.GingerLog.InitLog4Net();
 
+            if (System.Diagnostics.Trace.Listeners.Contains(CustomTraceListener.Instance) == false)
```

Comment:
Please add check for Debug log level, so it will only generate logs when Log level is set to debug

Reply:
Done